### PR TITLE
CTOR-1052 : Refresh cloud-aws-cloudwatch

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-cloudwatch

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 1]
```diff
@@ -2,44 +2,418 @@
 id: cloud-aws-cloudwatch
 title: Amazon CloudWatch
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 2]
```diff
@@ -2,44 +2,418 @@
...
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Privilèges AWS
+
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:73]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et secret key) et affectez-lui les privilèges suivants :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 3]
```diff
@@ -2,44 +2,418 @@
...
+* cloudwatch:getMetricStatistics
+* ec2:DescribeVpcs
+
+### Dépendances du Plugin
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:77]:
```suggestion
### Dépendances du plugin
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 4]
```diff
@@ -2,44 +2,418 @@
...
+
+1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
+3. Appliquez le modèle d'hôte **Cloud-Aws-CloudWatch-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro        | Description        | Valeur par défaut | Obligatoire |
+|:-------------|:-------------------|:------------------|:-----------:|
+| AWSACCESSKEY | Set AWS access key |                   |             |
+| AWSSECRETKEY | Set AWS secret key |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:200]: not mandatory?

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 5]
```diff
@@ -2,44 +2,417 @@
...
+
+1. Log into Centreon and add a new host through **Configuration > Hosts**.
+2. Fill in the **Name**, **Alias** & **IP Address/DNS** fields according to your resource's settings.
+3. Apply the **Cloud-Aws-CloudWatch-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
+4. Fill in the macros you want. Some macros are mandatory.
+
+| Macro        | Description        | Default value     | Mandatory   |
+|:-------------|:-------------------|:------------------|:-----------:|
+| AWSACCESSKEY | Set AWS access key |                   |             |
+| AWSSECRETKEY | Set AWS secret key |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:201]: not mandatory?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 6]
```diff
@@ -2,44 +2,418 @@
...
+```bash
+/usr/lib/centreon/plugins/centreon_aws_cloudwatch_api.pl \
+	--plugin=cloud::aws::cloudwatch::plugin \
+	--mode=get-metrics \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:261]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 7]
```diff
@@ -2,44 +2,418 @@
...
+que le connecteur arrive bien à superviser une instance AWS en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_aws_cloudwatch_api.pl \
+	--plugin=cloud::aws::cloudwatch::plugin \
+	--mode=get-metrics \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:258]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:200]:
```suggestion
| AWSACCESSKEY | Set AWS access key |                   | X           |
| AWSSECRETKEY | Set AWS secret key |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md (hunk 7 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:258]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md (hunk 5 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:201]:
```suggestion
| AWSACCESSKEY | Set AWS access key |                   | X           |
| AWSSECRETKEY | Set AWS secret key |                   | X           |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 8]
```diff
@@ -2,44 +2,417 @@
...
+```bash
+/usr/lib/centreon/plugins/centreon_aws_cloudwatch_api.pl \
+	--plugin=cloud::aws::cloudwatch::plugin \
+	--mode=get-metrics \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:260]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
	--aws-role-arn='' \
	--proxyurl=''  \
	--region='eu-west-1' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 9]
```diff
@@ -2,44 +2,418 @@
...
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+<TabItem value="Cloudwatch-Get-Metrics" label="Cloudwatch-Get-Metrics">
+
+| Nom    | Unité |
+|:-------|:------|
+| metric | N/A   |
+| metric | N/A   |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:62]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md [hunk 10]
```diff
@@ -2,44 +2,417 @@
...
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Cloudwatch-Get-Metrics" label="Cloudwatch-Get-Metrics">
+
+| Name   | Unit  |
+|:-------|:------|
+| metric | N/A   |
+| metric | N/A   |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch.md:61]:
```suggestion
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**